lib/erb_lint/linters/require_input_autocomplete.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'better_html'
+require 'better_html/tree/tag'
+
+module ERBLint
+  module Linters
+    class RequireInputAutocomplete < Linter
+      include LinterRegistry
+
+      TYPES_REQUIRING_AUTOCOMPLETE = [
+        "color",
+        "date",
+        "datetime-local",
+        "email",
+        "hidden",
```

Comment:
I think we wanted to remove 'hidden' from the list requiring autocomplete? cc: @gcornhill

Reply:
I'm getting errors for the form helper `:hidden_field_tag` and I see that the `hidden` tag is not removed too in main.
I think it should still be done.